Confus/Confus/Game.cpp

```diff
@@ -42,6 +42,15 @@ namespace Confus
 			irr::core::vector2df(0.56f, 0.0f), true);
     }
 
+    Game::~Game()
+    {
+        for(size_t i = 0u; i < m_PlayerArray.size(); i++)
+        {
+            //m_PlayerArray[i]->remove();
+            //delete(m_PlayerArray[i]);
+        }
```

Comment:
For me personally it would be a good reminder to keep it there so I can fix it later,. Just because it does not currently work does not mean it should be left out entirely.